client/components/jetpack-colophon/docs/example.js

```diff
@@ -0,0 +1,15 @@
+/** @format */
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import JetpackColophon from 'components/jetpack-colophon';
+
+export default function JetpackColophonExample() {
+	return <JetpackColophon />;
+}
+JetpackColophonExample.displayName = 'JetpackColophonExample';
```

Comment:
Looking at the example, it seems font size of the Colophon in DevDocs is too large (compared to the one at the bottom of the Stats page). How about specifically setting `font-size: 15px` there, so it can be consistent?

Haven't looked very far into it, but there might be discrepancies with vertical alignment and line height too, so let's make sure to compare them.

Reply:
👍  fe671d1aef0b6c63ad520d4cacc2e7f735332eac